pytest_html/plugin.py

```diff
@@ -344,14 +376,17 @@ def generate_summary_item(self):
             unicode_doc = unicode_doc.decode('utf-8')
         return unicode_doc
 
-    def _save_report(self, report_content, self_contained_html):
+    def _save_report(self, report_content):
         dir_name = os.path.dirname(self.logfile)
+        assets_dir = os.path.join(dir_name, 'assets')
         if not os.path.exists(dir_name):
             os.makedirs(dir_name)
+        if not os.path.exists(assets_dir):
+            os.makedirs(assets_dir)
```

Comment:
As this is beneath `dir_name` we only need to call `makedirs` on this, which will make the parent directory if needed.